dashboard/app/views/layouts/_header.html.haml

```diff
@@ -133,3 +133,6 @@
       #{stage_extras_enabled || 'null'}
     )
     //]]>
+
+:javascript
+  dashboard.header.renderProjectInfo(document.getElementsByClassName('project_info')[0]);
```

Comment:
using inline js and global vars here should not be necessary -- instead you could include a layouts/_header.js entry point which imports renderProjectInfo and then calls it directly.